dapps/multisig-toolkit/src/components/preview-effects/utils.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+export const onChainAmountToFloat = (amount: string, decimals: number) => {
+	if (!decimals) return undefined;
+	const total = parseFloat(amount);
+
+	return total / Math.pow(10, decimals);
+};
+
+export const formatAddress = (address: string) => {
```

Comment:
FYI I think @mysten/sui/utils exports something similar to this. Not a big deal to replicate just figured I'd let you know!

Reply:
Oh nvm this does last 10 chars